include/z64player.h

```diff
@@ -884,8 +884,8 @@ typedef enum PlayerCueId {
 #define PLAYER_STATE1_200        (1 << 9)
 // 
 #define PLAYER_STATE1_400        (1 << 10)
-// 
-#define PLAYER_STATE1_800        (1 << 11)
+// Currently carrying an actor
+#define PLAYER_STATE1_ACTOR_CARRY (1 << 11)
```

Comment:
I was wondering in oot if we wanted to go with `PLAYER_STATE1_CARRYING_ACTOR` instead. At least most of the flags i have named so far in oot are in "active" tense, as in it is what player is currently doing. But I think this is also fine 🤷

Reply:
`PLAYER_STATE1_CARRYING_ACTOR` does sound a little more natural to me, but what it is currently is also fine

Reply:
https://github.com/zeldaret/oot/pull/2213

Reply:
this change was accepted in oot, if you want to change it here as well

Reply:
oops seems like it already is changed